tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -580,6 +580,7 @@ public function unmountShare(string $user, string $itemId, string $shareSpaceId)
 	}
 
 	/**
+	 * @Given user :user has disabled sync for share :share using the Graph API
 	 * @When user :user unmounts share :share using the Graph API
```

Comment:
would be a mistake to put Given and When steps together. :eyes: 
Please, add separate step and do the response checks for Given step

Reply:
Will do